docs/user-docs/latest/plugins/strapi-plugins.md

```diff
@@ -0,0 +1,32 @@
+---
+title: List of Strapi plugins - Strapi User Guide
+description: Reference guide to Strapi plugins explaining how they work and how they expand a Strapi application
+---
+
+# List of Strapi plugins
+
+::: warning 🚧 This section of the user guide is a work in progress. Stay tuned!
+<br>
+:::
+
+No matter if they are installed by default, or additional, Strapi plugins allow to expand your application by adding more options and possibilities. All options are documented in their right places throughout the user guide. However, you can use the following documentation as a reference guide to know which Strapi plugins are available, how they work and which options they add to your Strapi application.
+
+## Additional plugins
+
+### <img width="28" src="../assets/plugins/icon_i18n-plugin.png"> Internationalization plugin
+
+The Internationalization plugin is an additional plugin that is not installed by default. It can be installed through the Marketplace (see [Installing plugins via the Marketplace](installing-plugins-via-marketplace.md)).
```

Comment:
⚠️  Warning:
 As said earlier, that might change.

Reply:
Thanks for the notice, Samuel !
I also added a [TODO in dev docs](https://github.com/strapi/documentation/pull/224/commits/cfd874e443d8029213c5bca952e44374ce8c7170) in case you decide to pre-install i18n with Strapi.